packages/web-common/src/gql/query-factories/block-website-sections.js

```diff
@@ -0,0 +1,32 @@
+const gql = require('graphql-tag');
+const defaultFragment = require('../fragments/block-website-sections');
+const { extractFragmentData } = require('../../utils');
+
+/**
+ * Builds the `BlockWebsiteSections` GraphQL operation.
+ *
+ * @param {object} params
+ * @param {string} [params.queryFragment] The `graphql-tag` fragment
+ *                                        to apply to the `websiteSections` query.
+ */
+module.exports = ({ queryFragment, queryName = '' } = {}) => {
+  const { spreadFragmentName, processedFragment } = extractFragmentData(queryFragment);
+  return gql`
+    query BlockWebsiteSections${queryName}($input: WebsiteSectionsQueryInput!) {
```

Comment:
This input doesn't have any required fields so should be an empty object by default: `($input: WebsiteSectionsQueryInput = {})`